backend/src/api/mutation.rs

```diff
@@ -285,6 +291,39 @@ impl Mutation {
             target_realm
         };
 
+        let series = if let Some(current_realm_path) = current_realm_path {
+            let old_realm = Realm::load_by_path(current_realm_path, context)
+                .await?
+                .ok_or_else(|| invalid_input!("`currentRealmPath` does not refer to a valid realm"))?;
+            let blocks = BlockValue::load_for_realm(old_realm.key, context).await?;
+            if blocks.len() > 1 {
+                // This is already checked and prohibited on the front end side (admin UI),
+                // but I suppose checking here doesn't hurt .
+                return Err(invalid_input!("series can only be moved if it's current realm has no other blocks"));
+            }
+
+            if old_realm.number_of_descendants(context).await? < 1 {
+                // The realm has no children, so it can be removed.
+                Realm::remove(old_realm.id(), context).await?;
+            } else {
+                // At this point we can be certain that there is only one block, which is the series block.
+                // Ideally we would restore the previous title, but that's not stored anywhere. So the realm
+                // gets the name of its path segment.
+                Realm::rename(
+                    old_realm.id(),
+                    UpdatedRealmName::plain(old_realm.path_segment),
+                    context,
+                ).await?;
```

Comment:
Uh mhhhh. That's not super nice? The path segment isn't great. Is maybe using the series title better? 

How did you decide that we want to allow this case anyway? The case where the realm has children. I don't remember anymore what we said.

Reply:
Yeah you and Julian said not to allow it, but that's kinda mean since _it is_ allowed to mount series in realms with children realms. So if anyone makes a mistake doing that, I feel it should be possible to correct that.